### DISCUSSION TITLE
Styles issue Diamond issue

### DISCUSSION THREAD

**Discussion by gongora92** [primefaces/community#1195] [PrimeVue Templates]
I just got Diamond PrimeVue 5.2 and tried to run it locally
I run `yarn install ` and  `sass --update public/layout/:public/layout/` and after `yarn dev` but it seems the styles are broken
<img width="1425" alt="Screenshot 2024-02-14 at 13 19 31" src="https://github.com/primefaces/community/assets/44581880/996763ab-fc29-4915-9c3a-3e70d9796c7e">

**Comment by tugcekucukoglu** [comment]
Could you try to use the latest 5.3.0 version?
